examples/source_examples/timescale_db/README.md

```diff
@@ -0,0 +1,128 @@
+# TimescaleDB Example Connector
+
+## Connector overview
+
+The TimescaleDB connector allows you to extract time-series data and vector data from a TimescaleDB database and load it into your destination. This connector is designed to handle vector data efficiently using incremental sync patterns with proper checkpointing.
+
+## Requirements
+
+* [Supported Python versions](https://github.com/fivetran/fivetran_connector_sdk/blob/main/README.md#requirements)   
+* Operating system:
+  * Windows: 10 or later (64-bit only)
+  * macOS: 13 (Ventura) or later (Apple Silicon [arm64] or Intel [x86_64])
+  * Linux: Distributions such as Ubuntu 20.04 or later, Debian 10 or later, or Amazon Linux 2 or later (arm64 or x86_64)
+
+## Getting started
+
+Refer to the [Setup Guide](https://fivetran.com/docs/connectors/connector-sdk/setup-guide) to get started.
+
+## Features
+
+1. Connects to TimescaleDb database using `psycopg2`
```

Comment:
I don't think a numbered list is needed and appropriate for the feature list, make it a bulleted one.